CONTRIBUTING.md

```diff
@@ -56,6 +56,9 @@ Feel free to copy the `.circleci/` directory into your example's root directory.
 Once approved, we will make sure that your example is set up on our CircleCI
 account.
 
+To ensure your example runs in [GitHub Codespaces](https://docs.github.com/en/codespaces/overview),
+copy the `.devcontainer/` directory into your example's root directory.
```

Comment:
```suggestion
copy the [.devcontainer/](.devcontainer/) directory into your example's root directory.
```

I don't have time to test the above before my 1:00 meeting but I mean: can you make this a link instead?

Reply:
I was aiming for consistency with the line above, but it makes sense. I'll update the line above as well.